modules/dnn/test/test_tf_importer.cpp

```diff
@@ -26,6 +26,21 @@ static std::string _tf(TString filename)
     return (getOpenCVExtraDir() + "/dnn/") + filename;
 }
 
+CV_ENUM(DNNTarget, DNN_TARGET_CPU, DNN_TARGET_OPENCL)
```

Comment:
Multiple "CV_ENUM()" declarations under the same namespace::name should be avoided (to avoid cases with different values - linker will pick just one and will not warn about duplicate names because they are templates).
You should use anonymous namespace everywhere to avoid compiler mess with these symbols.
Or move this declaration and generator below into `test_precomp.hpp`.

Reply:
Replaced all DNNTarget and DNNBackend from tests into `test_precomp.hpp`.